test/functional/change_stream_tests.js

```diff
@@ -1282,30 +1282,32 @@ describe('Change Streams', function() {
         var theCollection = theDatabase.collection('pipeTest');
         var thisChangeStream = theCollection.watch(pipeline);
 
-        var filename = '/tmp/_nodemongodbnative_stream_out.txt';
-        var outStream = fs.createWriteStream(filename);
+        var outStream = new stream.PassThrough({ objectMode: true });
```

Comment:
nit: can we use `const` for the new code?

Reply:
oops, forgot to do this. Put up https://github.com/mongodb/node-mongodb-native/pull/2056 to fix this.